### PR TITLE
[VL][CI] Disable run_setup_script in building velox

### PR DESCRIPTION
**Pull request by PZD-CHINA**
set run_setup_scipts=OFF as all dependencies are already installed
 
 to solve this comment: https://github.com/oap-project/gluten/pull/4348/files#r1448402406

### CODE REVIEW

File: .github/workflows/velox_be.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ jobs:
           docker exec ubuntu2004-test-$GITHUB_RUN_ID bash -c '
           cd /opt/gluten/ep/build-velox/src && \
           ./get_velox.sh --velox_home=/opt/velox && \
-          ./build_velox.sh --velox_home=/opt/velox --enable_ep_cache=ON --build_test_utils=ON'
+          ./build_velox.sh --run_setup_scipts=OFF --velox_home=/opt/velox --enable_ep_cache=ON --build_test_utils=ON'
```
Comment (PHILO-HE) [inline .github/workflows/velox_be.yml:66]: Typo: 
run_setup_script

**Comment by PHILO-HE** [comment]
@PZD-CHINA, I note CI reported errors for azure storage. We may need to set `--enable_abfs=ON` in `dockerfile-build` to pre-install required dependencies when building docker image for azure storage. We can do that in a separate PR and then update docker image.
See centos's dockerfile-build:
https://github.com/oap-project/gluten/blob/main/tools/gluten-te/centos/dockerfile-build#L61

**Comment by PHILO-HE** [comment]
I think we can remove `RUN_SETUP_SCRIPT` from build summary to directly skip the build if it's the only difference, as in docker image we set it ON, but in CI workflow we are setting it OFF. cc @zhztheplayer 
https://github.com/oap-project/gluten/blob/main/ep/build-velox/src/build_velox.sh#L182

**Comment by zhztheplayer** [comment]
> I think we can remove RUN_SETUP_SCRIPT from build summary

Yes. We don't have mechanism to detect this level of build reuse (e.g., ENABLE_HDFS=OFF could reuse ENABLE_HDFS=ON, or RUN_SETUP_SCRIPT=OFF could reuse RUN_SETUP_SCRIPT=ON). Currently we can just remove it from build summary and it's fine.

**Comment by PZD-CHINA** [comment]
> @PZD-CHINA, I note CI reported errors for azure storage. We may need to set `--enable_abfs=ON` in `dockerfile-build` to pre-install required dependencies when building docker image. We can do that in a separate PR and then update docker image. See centos's dockerfile-build: https://github.com/oap-project/gluten/blob/main/tools/gluten-te/centos/dockerfile-build#L61

I raised a PR [# 4413](https://github.com/oap-project/gluten/pull/4413) to solve this problem.

**Review by zhouyuan — APPROVED** [review]
👍

**Pull request merged**